src/main/java/com/team4/testingsystem/services/FileAnswerService.java

```diff
@@ -0,0 +1,64 @@
+package com.team4.testingsystem.services;
+
+import com.team4.testingsystem.entities.FileAnswer;
+import com.team4.testingsystem.exceptions.NotFoundException;
+import com.team4.testingsystem.repositories.FileAnswerRepository;
+import com.team4.testingsystem.repositories.QuestionRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.NoSuchElementException;
+import java.util.logging.Logger;
+import java.util.logging.Level;
+
+@Service
+public class FileAnswerService {
+    private final Logger logger = Logger.getLogger(FileAnswerService.class.getName());
+    private final FileAnswerRepository fileAnswerRepository;
+    private final QuestionRepository questionRepository;
+
+    @Autowired
+    public FileAnswerService(FileAnswerRepository fileAnswerRepository, QuestionRepository questionRepository) {
+        this.fileAnswerRepository = fileAnswerRepository;
+        this.questionRepository = questionRepository;
+    }
+
+    public void create(long id, String url, long questionId) {
+        FileAnswer fileAnswer = new FileAnswer();
+        try {
+            fileAnswer.setId(id);
+            fileAnswer.setUrl(url);
+            fileAnswer.setQuestion(questionRepository.findById(questionId).get());
+            fileAnswerRepository.save(fileAnswer);
```

Comment:
Creating an object and use lots of setters to its fields seems weird. Please, create a builder or an appropriate constructor instead.